lib/combustion.rb

```diff
@@ -28,14 +30,14 @@ module Configurations
 
   AVAILABLE_MODULES = begin
     keys = MODULES.keys
-    version = Rails.version.to_f
-
-    keys.delete(:sprockets) unless (3.1..6.1).include?(version)
-    keys.delete(:active_job) unless version >= 4.2
-    keys.delete(:action_cable) unless version >= 5.0
-    keys.delete(:active_storage) unless version >= 5.2
-    keys.delete(:action_text) unless version >= 6.0
-    keys.delete(:action_mailbox) unless version >= 6.0
+    gate = VersionGate.new("rails")
+
+    keys.delete(:sprockets) unless gate.call(">= 3.1", "<= 6.1")
+    keys.delete(:active_job) unless gate.call(">= 4.2")
+    keys.delete(:action_cable) unless gate.call(">= 5.0")
+    keys.delete(:active_storage) unless gate.call(">= 5.2")
+    keys.delete(:action_text) unless gate.call(">= 6.0")
+    keys.delete(:action_mailbox) unless gate.call(">= 6.0")
```

Comment:
personally for clarity I'd call the variable `rails_gate` etc.

Reply:
Good idea, I've made this change.